### PR TITLE
Release 0.20.0

### PR DESCRIPTION
**Pull request by ConnorRigby**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.20.0
+
+* Potential Breaking Changes
+  * this realease includes a change to how CA certificates are distributed.
+    This is mostly internal, However if you self host your own instance of
+    NervesHub you will need to follow the steps in
+    [nerves_hub_ca_store](https://github.com/nerves_hub/nerves_hub_ca_store)
```
Comment (jjcarstens) [inline CHANGELOG.md:9]:
```suggestion
* Breaking Changes
  * This release includes a change to how CA certificates are used in the connection
    to NervesHub. If you are connecting to the publicly hosted https://nerves-hub.org,
    then no changes are required.
    If you are manually supplying `:ca_certs` config value to connect to another instance
    of NervesHub, then you will need to update you config following the instructions of
    [CA Certificates](README.md#ca-certificates) in the README
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
 
+## 0.20.0
+
+* Breaking Changes
+  * This release includes a change to how CA certificates are used in the connection
+    to NervesHub. If you are connecting to the publicly hosted https://nerves-hub.org,
+    then no changes are required.
+    If you are manually supplying `:ca_certs` config value to connect to another instance
+    of NervesHub, then you will need to update you config following the instructions of
+    [CA Certificates](README.md#ca-certificates) in the README
```
Comment (fhunleth) [inline CHANGELOG.md:11]:
```suggestion
    [CA Certificates](https://hexdocs.pm/nerves_hub_link/readme.html#ca-certificates) in the README.
```

That hyperlink should work when it's published.

**Review by fhunleth — APPROVED** [review]

**Review by jjcarstens — APPROVED** [review]

**Comment by mobileoverlord** [comment]
Did I miss a conversation? We are going from 0.10 to 0.20. Why not 0.11?

**Comment by ConnorRigby** [comment]
> Did I miss a conversation? We are going from 0.10 to 0.20. Why not 0.11?

it was sort of an arbitrary decision by me. I think 0.11 is equally as valid.

**Comment by ConnorRigby** [comment]
Closed in favor of #85

**Pull request closed without merging**